specification/gedcom-3-structures-3-meaning.md

```diff
@@ -907,28 +907,55 @@ As of version 7.0, only 2 media types are supported by this structure:
-However, exporting must only use the core XML entities, translating any other entities into their corresponding Unicode characters.
-:::
+    <div class="note">
+    Applications are welcome to support more XML entities or HTML character references in their user interface.
+    However, exporting must only use the core XML entities, translating any other entities into their corresponding Unicode characters.
+    </div>
 
-:::note
-Applications are welcome to support additional HTML elements,
-but they should ensure that content is meaningful if those extra elements are ignored and only their content text is displayed.
-:::
+    <div class="note">
+    Applications are welcome to support additional HTML elements,
+    but they should ensure that content is meaningful if those extra elements are ignored and only their content text is displayed.
+    </div>
 
+    If needed, `text/html` can be converted to `text/plain` using the following steps:
 
-:::note
-Media types are also used by external files, as described under `FORM`. External file media types are not limited to `text/plain` and `text/html`.
-:::
+    1. Replace any sequence of 1 or more spaces, tabs, and line breaks with a single space
+    2. Case-insensitively replace each `<p`...`>`, `</p`...`>`, and `<br`...`>` with a line break
+    3. Remove all other `<`...`>` tags
+    4. Replace each `&lt;` with `<` and `&gt;` with `>`
+    5. Replace each `&amp;` with `&`
+
+- `text/markdown` employs a plain text format for creating structured documents, and is designed
+  to be fairly understandable by users even if only displayed as plain text.
+  Applications should support at least the following:
 
-If needed, `text/html` can be converted to `text/plain` using the following steps:
+    - `**`, `*`, `__`, and `~~` for bold, italic, underlined, and strike-through text.
+    - `#` Heading element for Heading 1 (placed on its own line).
+    - `##` Heading element for Heading 2 (placed on its own line).
+    - `*` List element for Bullet List (each item in a bulleted list on its own line).
+    - `1.` List element for Numbered List (a number followed by an item, each on its own line).
 
-1. Replace any sequence of 1 or more spaces, tabs, and line breaks with a single space
-2. Case-insensitively replace each `<p`...`>`, `</p`...`>`, and `<br`...`>` with a line break
-3. Remove all other `<`...`>` tags
-4. Replace each `&lt;` with `<` and `&gt;` with `>`
-4. Replace each `&amp;` with `&`
+    For example:
+
+    <div class="example">
+
+    ````gedcom
+    1 NOTE # 1940 US Census for New York, New York
+    2 CONT __Individuals located at this address__
+    2 CONT * John **Doe** (age 35)
+    2 CONT * Sally (age 32)
+    2 CONT * Junior (age 3)
+    2 CONT * Betty (age 1)
+    2 MIME text/markdown
+    ````
+    
+    </div>
+
+    Supporting more of [Markdown](https://commonmark.org) is encouraged.
```

Comment:
Discussed in steering committee 18 MAR 2025

The listed subset of commonmark is small, relative to the spec as a whole (omitting at least block quotes, inline code, code blocks, hard breaks, H3 through H6, underlined header syntax, inline HTML, horizontal rules, images, links, escaping characters), and lacks the clarifications about syntactic ambiguities that commonmark has (though some of those are introduced by things we omit).
- Should we encourage supporting all of CommonMark or just the provided subset?
- Should we keep the subset but reference CommonMark for syntactic ambiguity?
- Should we remove the subset and simply reference CommonMark?
- Should we say anything about supporting CommonMark extensions (like strike-through or definition lists)?

Reply:
Updated